prettier.config.js

```diff
@@ -1,14 +1,11 @@
-/** @type {import("prettier").Config} */
-const config = {
+module.exports = {
```

Comment:
Is this a merge conflict?

Reply:
It might be, I updated this file because I realized I didn't have the prettier configured to what Ryan posted in the survivor-app channel, but I'll revert this change.